release/src/Upload.hs

```diff
@@ -235,7 +235,7 @@ publishStagingRepo baseRequest manager comDamlRepoId comDigitalassetRepoId = do
          $ setRequestPath  "/service/local/staging/bulk/promote"
          $ setRequestHeader "content-type" [ "application/json" ]
          $ setRequestHeader "accept" [ "application/json" ]
-         $ setRequestBodyLBS (textToLazyByteString $ "{\"data\":{\"stagedRepositoryIds\":[\"" <> comDamlRepoId <> "\",\"" <> comDigitalassetRepoId <> "]\",\"description\":\"\",\"autoDropAfterRelease\":true}}") baseRequest
+         $ setRequestBodyLBS (textToLazyByteString $ "{\"data\":{\"stagedRepositoryIds\":[\"" <> comDamlRepoId <> "\",\"" <> comDigitalassetRepoId <> "\"],\"description\":\"\",\"autoDropAfterRelease\":true}}") baseRequest
```

Comment:
You might just want to switch to `aeson` for doing the JSON encoding to avoid that kind of error prone string munging.

Reply:
I will, if this release round doesn't work!